.github/workflows/snapcraft.yml

```diff
@@ -3,6 +3,7 @@ name: Publish snap package
 on:
   release:
     types: [published]
+  workflow_dispatch:
```

Comment:
Do we need to enable manually triggered publish?

Reply:
Nope, I'll remove.